web_create_write_confirm/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_popup_message,popup.message,model_popup_message,,1,1,1,1
```

Comment:
With these rights even portal user can create, update, delete records. Everyone should be able to read but only admin or erp manager should be able create, update, delete.